decoder/pokemon.go

```diff
@@ -655,6 +655,19 @@ func (pokemon *Pokemon) updateSpawnpointInfo(ctx context.Context, db db.DbDetail
 		panic(err)
 	}
 
+	log.Infof("Encounter [START] %s: Spawnpoint %d Timestamp %d ExpireTimestampVerified: %t ExpireTimestamp: %v", pokemon.Id, spawnId, timestampMs, pokemon.ExpireTimestampVerified, pokemon.ExpireTimestamp)
+
+	if timestampMs == 0 {
```

Comment:
I checked usages and this timestamp might be never 0. On encounter we pass now() :)

Reply:
In which case that needs to be changed (I thought I did) so that this logic can work correctly.